apps/cloudappclient/manager.js

```diff
@@ -13,9 +13,18 @@ function Manager (exe, Skill) {
   this.Skill = Skill
   this.skills = []
   this.isAppActive = true
+  this.manualPause = false
```

Comment:
This variable `manualPause` belongs to a specific product and needs to be annotated to explain the role of this variable.